### PR TITLE
📖 update install doc (go install)

### PR DESCRIPTION
**Pull request by Huang-Wei**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

`go install` a LATEST version to ease doc maintenance.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by Huang-Wei** [comment]
@wzshiming one weird symptom I found when `go install ...@latest`:

```
⇒  go install sigs.k8s.io/kwok/cmd/{kwok,kwokctl}@latest
...
⇒   ~/go/bin/kwok --version
kwok version v0.3.0-alpha go1.20.4 (darwin/arm64)
⇒   ~/go/bin/kwokctl --version
kwokctl version v0.3.0-alpha go1.20.4 (darwin/arm64)
```

why it has a `-alpha` postfix?

**Comment by wzshiming** [comment]
https://github.com/kubernetes-sigs/kind/blob/3610f606516ccaa88aa098465d8c13af70937050/pkg/cmd/kind/version/version.go#L31
This is a reference to the version number of Kind, which adds a suffix to binaries that are not released and not built from the same commit.

kwokctl is a bit different from Kind, that it relies on the same version of the kwok image, so its latest build may not be directly usable, which is why my previous documentation had a version number

File: site/content/en/docs/user/installation.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ brew install kwok
 also, you can install `kwokctl` and `kwok` binaries via [golang].
 
 ``` bash
-go install sigs.k8s.io/kwok/cmd/{kwok,kwokctl}@v0.2.0
+go install sigs.k8s.io/kwok/cmd/{kwok,kwokctl}@latest
```
[the remark this answers is not in the document]
Comment (wzshiming) [reply site/content/en/docs/user/installation.md:30]: Maybe we can get the latest version number as this, WDYT?
https://kwok.sigs.k8s.io/docs/user/kwok-in-cluster/#variables-preparation

[the remark this answers is not in the document]
Comment (Huang-Wei) [reply site/content/en/docs/user/installation.md:30]: Sounds good, let me update.

**Comment by Huang-Wei** [comment]
> [kubernetes-sigs/kind@`3610f60`/pkg/cmd/kind/version/version.go#L61](https://github.com/kubernetes-sigs/kind/blob/3610f606516ccaa88aa098465d8c13af70937050/pkg/cmd/kind/version/version.go#L61)
> This is a reference to the version number of Kind, which adds a suffix to the version of binaries that are not released and not built from the same commit.

I'm not sure I fully understand this. For installation via `go install`, I don't fully get it why it carries the alpha postfix - even when I install via `go install ...@v0.3.0`.

**Comment by wzshiming** [comment]
ohh it looks like I forgot to remove the suffix when I made the commit for the 0.3 tag.

**Review by wzshiming — APPROVED** [review]
/approve
/lgtm

**Pull request merged**